web-ui/src/main/resources/catalog/templates/admin/settings/sources.html

```diff
@@ -234,6 +234,27 @@
               <p class="help-block" data-translate="">subPortalGroupOwnerHelp</p>
             </div>
 
+            <label data-translate="">datahubEnabled</label>
+            <input
+              type="checkbox"
+              name="datahubEnabled"
+              id="datahubEnabled"
+              data-ng-model="source.datahubEnabled"
+            />
+            <p class="help-block" data-translate="">sourceDatahubEnabled-help</p>
```

Comment:
All the translation keys need to be added to the translation file here: https://github.com/geonetwork/core-geonetwork/blob/main/web-ui/src/main/resources/catalog/locales/en-admin.json